internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -304,6 +324,15 @@ func (s *SchedulerManager) DeleteScheduler(ctx context.Context, schedulerName st
 	return nil
 }
 
+func (s *SchedulerManager) getScheduler(ctx context.Context, schedulerManager string) (*entities.Scheduler, error) {
```

Comment:
Why don't you add the scheduler cache logic to the "GetActiveScheduler" instead of creating a new method? I have the impression we have A LOT of "getScheduler" methods for the scheduler manager

Reply:
Also, why did you put the name of the parameter "schedulerManager" instead of "schedulerName"? It is a typo?

Reply:
@caiordjesus  By doing this, we would create a lateral dependency between the methods and we already had trouble with it in the past... the right thing would be to change the GetActiveScheduler to use this new method I've created, what do you think?

Reply:
[fa898a4](https://github.com/topfreegames/maestro/pull/484/commits/fa898a4d4601a9280bd9b7e4554bdec248b5e3db)